README.md

```diff
@@ -155,16 +158,22 @@ Monitoring resources creation is automatically triggered via separate pipeline,
 
 ### Monitoring
 
-Azure Monitoring resources like Log Analytics and Grafana are created by the **Build Monitoring** pipeline. This workflow is automatically triggered by the **Build VM** pipeline but you have the chance to manually run it as well.
+Azure Monitoring resources like Log Analytics and Grafana are created by the **Build Monitoring** pipeline. This workflow is automatically triggered by the **Build VM** pipeline.
 You can also run the **Build Monitoring** workflow manually to create monitoring resources for an existing cluster.
 
+It deploys the [Azure Monitor Extension](https://learn.microsoft.com/en-us/azure/azure-monitor/containers/kubernetes-monitoring-enable?tabs=cli#arc-enabled-cluster) for Arc-enabled clusters and the [Azure Monitor Agent](https://learn.microsoft.com/en-us/azure/azure-monitor/agents/agents-overview) as extension for VMs.
+
+Additionally, the Windows OS images are prepared with the Windows Assessment and Deployment Kit ([Windows ADK](https://learn.microsoft.com/en-us/windows-hardware/get-started/adk-install)) to expose the host memory metrics to the Log Analytics workspace via DataCollection Rules. On Linux those metrics are available out-of-the-box in the file system of the OS itself.
+
```

Comment:
Is this the case for the HCI box?

Reply:
It looks like for HCI box, you don't get the granularity w/ per process snapshots since it doesn't use the images that are build from the pipeline

Reply:
That is a really good question:
this is also the case for HCIBox...that is the reason I added the azure_jumpstart_hcibox/artifacts/PowerShell/Install-HostMemCollector.ps1 file that sets up the ADK stuff and scheduler to write the memory output files that are used via the dataCollectionRule

Reply:
Ohh nice! Didn't see that.